src/EnergyExporter/MQTT/MqttExporter.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using EnergyExporter.Devices;
+using EnergyExporter.InfluxDb;
+using EnergyExporter.Options;
+using EnergyExporter.Services;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using MQTTnet;
+using MQTTnet.Client;
+using MQTTnet.Protocol;
+
+namespace EnergyExporter.MQTT;
```

Comment:
Per convention, I think `EnergyExporter.Mqtt` would be more correct (also rename directory please).